src/scripts/modules/add_destination.js

```diff
@@ -0,0 +1,105 @@
+import destinationsList from "../destinationList.js";
+import events from "./pubsub.js";
+
+let form;
+let radioBtns;
+let radioVisited;
+let radioBucketlist;
+let cityInput;
+let countryInput;
+let photoInput;
+let storedCity;
+//new list item state before the item is added
+let formState = {
+  city: null,
+  country: null,
+  photo: null,
+};
+
+function cacheDom() {
+  form = document.getElementById("add-city-form");
+  uploadBtn = document.getElementById("upload");
```

Comment:
The  `uploadBtn` is not needed, so you can delete it. Pay attention to the console while you develop, you'll notice the error there ;)